### PR TITLE
Add current user to navbar

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**

### CODE REVIEW

File: web/components/Navbar.tsx [hunk 1]
```diff
@@ -23,8 +23,10 @@ const useStyles = makeStyles({
   },
 });
 
-type NavbarProps = {};
-export const Navbar: React.FC<NavbarProps> = () => {
+type NavbarProps = {
+  user: any;
```
Comment (jenny-zhong) [inline web/components/Navbar.tsx]: are we going to declare types down the line?

File: web/components/Navbar.tsx [hunk 2]
```diff
@@ -23,8 +23,10 @@ const useStyles = makeStyles({
...
+};
+export const Navbar: React.FC<NavbarProps> = (props) => {
```
Comment (jenny-zhong) [inline web/components/Navbar.tsx]:
```suggestion
export const Navbar: React.FC<NavbarProps> = ({ user }) => {
```
we can destructure our props so we don't need to say `props.something` everytime we want to use our props

File: web/components/Navbar.tsx [hunk 3]
```diff
@@ -43,6 +45,11 @@ export const Navbar: React.FC<NavbarProps> = () => {
           <Link to="/give">
             <MenuTab name="Giving" pages={[]}></MenuTab>
           </Link>
+          {props.user && (
+            <Link to="/profile">
+              <MenuTab name={props.user.username} pages={[]}></MenuTab>
```
Comment (jenny-zhong) [inline web/components/Navbar.tsx]:
```suggestion
              <MenuTab name={user.username} pages={[]}></MenuTab>
```

File: web/components/Navbar.tsx [hunk 4]
```diff
@@ -43,6 +45,11 @@ export const Navbar: React.FC<NavbarProps> = () => {
           <Link to="/give">
             <MenuTab name="Giving" pages={[]}></MenuTab>
           </Link>
+          {props.user && (
```
Comment (jenny-zhong) [inline web/components/Navbar.tsx]:
```suggestion
          {user && (
```

File: web/components/Navbar.tsx (hunk 1 above)
Comment (Kyle-Verhoog) [reply web/components/Navbar.tsx]: hmm. I think in order to get these types we'd need to generate them for the graphql schema. Maybe we should tackle that first. WDYT?

Comment (Kyle-Verhoog) [reply web/components/Navbar.tsx]: Done in #34

**Pull request merged**